File: Models/Quiz/QuizAnswer.cs
using System.ComponentModel.DataAnnotations;

namespace quizmoon.Models
{
    public class QuizAnswer
    {
        public long Id { get; set; }

        [MaxLength(50, ErrorMessage = "Quiz answer text cannot be longer than 50 characters")]
        public string Text { get; set; }
        public byte[] Image { get; set; }
        public bool Correct { get; set; }

        public QuizQuestion QuizQuestion { get; set; }
        public long QuizQuestionId { get; set; }
    }
}

File: ClientApp/src/_models/quiz/QuizAnswer.ts
export class QuizAnswer {
    id: number;
    text: string;
    image?: File;
    correct: boolean;
    textType?: boolean;
    dirty: boolean = false;
    type: string = "Text";

    constructor(id: number, text: string, correct: boolean) {
        this.id = id;
        this.text = text;
        this.correct = correct;
        this.image = null;
        this.textType = null;
    }
}